src/Compilers/Core/Desktop/AnalyzerFileReference.cs

```diff
@@ -208,13 +213,12 @@ internal void AddAnalyzers(ImmutableArray<DiagnosticAnalyzer>.Builder builder, s
                 // If there are no analyzers, don't load the assembly at all.
                 if (!analyzerTypeNameMap.ContainsKey(language))
                 {
-                    this.AnalyzerLoadFailed?.Invoke(this, new AnalyzerLoadFailureEventArgs(AnalyzerLoadFailureEventArgs.FailureErrorCode.NoAnalyzers, null, null));
```

Comment:
@heejaechang If you have a type that claims to be an analyzer but we can't load it for whatever reason, yes, we still report a warning.